src/python/director/screengrabberpanel.py

```diff
@@ -27,9 +27,11 @@ def __init__(self, widgets):
 
 
 class ScreenGrabberPanel(object):
+    # HACK
+    instance = None
 
     def __init__(self, view):
-
+        ScreenGrabberPanel.instance = self
```

Comment:
Of course, please feel free to leave open and update me when you want.  Maybe I'll do some work on this, too.  I'm not using LCM in my current project but I am using the screen recorder with other replay mechanisms, and I definitely want to add more features to support programmable video recording.